python/suricata/sc/suricatasc.py

```diff
@@ -1,5 +1,5 @@
-#!/usr/bin/python
-# Copyright(C) 2012-2020 Open Information Security Foundation
+#!/usr/bin/env python
+# Copyright(C) 2012-2023 Open Information Security Foundation
```

Comment:
Same here.

Looks like these are both executable in current install, which is probably a mistake as well.

Reply:
Sounds fair. I have removed the x bit in the install in the newer version of this PR, please check.